### DISCUSSION TITLE
Multiple windows workers on single machine

### DISCUSSION THREAD

**Discussion by grantith** [windmill-labs/windmill#4794] [Questions] [answered]
Can we setup multiple windows workers on a single machine? Would we simply deploy the executable to multiple directories?

https://www.windmill.dev/docs/misc/windows_workers

**Comment by rubenfiszel** [comment, marked as answer]
Yes, you can just call the same executable multiple times without a need for different dirs, every worker specific files will be in the `tmpfolder/windmill/wk-xxx/`. For log purposes, you might need to set FORCE_HOSTNAME=x that is different for each executable launch
